api/services/EmailService.js

```diff
@@ -39,16 +37,11 @@ const EmailService = {
     },
 
     _sendRequestToNotificationService(postData, url) {
-        emailRequest(
-            { url, body: postData },
-            (err, res, body) => {
-                if (err) {
-                    sails.log.error(err);
-                } else {
-                    sails.log.info(res.statusCode, body);
-                }
-            }
-        );
+        emailRequest.post(url,{ data: postData }).then((response) => {
```

Comment:
is url always the same?

Reply:
Nope, it will either be '/confirm-submission' or '/failed-documents' based on the function that is called.